docs/csharp/whats-new/csharp-11.md

```diff
@@ -0,0 +1,84 @@
+---
+title: What's new in C# 11 - C# Guide
+description: Get an overview of the new features coming in C# 11.
+ms.date: 03/04/2022
+---
+# What's new in C# 11
+
+Beginning with the .NET 6.0.200 SDK or Visual Studio 2022 version 17.1, preview features in C# are available for you to try.
+
+> [!IMPORTANT]
+> These are currently preview features. You must [set `<LangVersion>` to `Preview`](../language-reference/compiler-options/language.md#langversion) to enable these features. Any feature may change before its final release. These features may not all be released in C# 11. Some may remain in a preview phase for longer based on feedback on the feature.
```

Comment:
nit: We typically use lower-cased "preview" for `<LangVersion>`

Reply:
"These are currently preview features. You must ..." is duplicated. Looks like a copy&paste mistake.